### DISCUSSION TITLE
Version 3.2.7 Running on Windows - Download button

### DISCUSSION THREAD

**Discussion by gj2355** [PodcastGenerator/PodcastGenerator#712] [Q&A] [answered]
Hello,

Loving the latest version of PodcastGenerator

One thing I have found is that when I click the download button the podcast plays. It doesn't download. I've tried different browsers with the same result.

Is that because it's my own podcast, and perhaps it will download for a visitor? Or have I done something wrong? 

Thanks

**Comment by charabaruk** [comment, marked as answer]
You found a bug in the theme! The download buttons are missing an attribute that tells web browsers to download the episode instead of opening it in the browser. I'll track this with #714.
